config/peerpods/podvm/azure-podvm-image-cm.yaml

```diff
@@ -59,3 +59,18 @@ data:
 
   # Custom Agent Policy
   #AGENT_POLICY: "" # set to base64 encoded agent policy
+
+  # bootc
+  #PODVM_IMAGE_URI: bootc::image-registry.openshift-image-registry.svc:5000/openshift-sandboxed-containers-operator/podvm-bootc
+  # Custom bootc build configuration: https://osbuild.org/docs/bootc/#-build-config
```

Comment:
Do we want to expose the disk config to user or have a fixed config which is tested and supported ?

Reply:
yes, indeed we can have a default config file